### PR TITLE
Adding JSON

### PR DESCRIPTION
**Pull request by hhan-solvd**
Created one JSON file for at least five classes from the hierarchy;
Added Jackson’s annotation to the hierarchy;
Parsed JSON using Jackson.

### CODE REVIEW

File: src/main/java/com/solvd/app/json/JSONUtils.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            String className = object.getClass().getSimpleName();
+            objectMapper.writeValue(new File(System.getProperty("user.dir") +
+                    "/src/main/resources/json/" + className + ".json"), object);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static <T> T readJSON(String jsonFilePath, Class<T> targetType) {
+        ObjectMapper objectMapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
```
Comment (ankochergin) [inline src/main/java/com/solvd/app/json/JSONUtils.java]: we don't need formatting feature for reading from the file

File: src/main/java/com/solvd/app/json/JSONUtils.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import java.io.File;
+import java.io.IOException;
+
+public class JSONUtils {
+
+    public static <T> void writeJSON(T object) {
```
Comment (ankochergin) [inline src/main/java/com/solvd/app/json/JSONUtils.java]: maybe we should put path as method parameter too? what do you think? and the same thing for jaxb

File: src/main/java/com/solvd/app/json/JSONUtils.java (hunk 1 above)
Comment (hhan-solvd) [reply src/main/java/com/solvd/app/json/JSONUtils.java]: Done.

File: src/main/java/com/solvd/app/json/JSONUtils.java (hunk 2 above)
Comment (hhan-solvd) [reply src/main/java/com/solvd/app/json/JSONUtils.java]: Yes, I agree with you. I just updated my code accordingly.

File: src/main/resources/json/Prescription.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{
+  "prescriptionDate" : "2023-06-05",
+  "prescription_id" : 12,
```
Comment (ankochergin) [inline src/main/resources/json/Prescription.json]: please use the same style for keys in your json. you used both camel and snake styles

Comment (hhan-solvd) [reply src/main/resources/json/Prescription.json]: Just fixed it.

**Pull request merged**